### ISSUE TITLE
Documentation Improvements

### ISSUE THREAD

**Issue by jarretraim** [skylinerhq/hello-rails#1]
It would be nice to have docs for this example that explain the differences between a basic Rails app and one that works with Skyliner. As of now, I have to go through the commits to this repo by hand, some of which include major changes that are hard to follow. 

Lack of these docs also means that it is much more difficult to port an existing Rails app over or maintain my own app with up to date versions, etc.

**Comment by codahale** [comment]
Thanks for pointing this out — we used to have Rails-specific documentation in Skyliner, but recently changed that and this got lost in the process.
